### PR TITLE
Followed hashtags view improvements

### PR DESCRIPTION
**Pull request by Tak**
Add click and longpress handler to the followed hashtags view

#4459

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt [hunk 1]
```diff
@@ -180,6 +184,15 @@ class FollowedTagsActivity :
         return viewModel.searchAutocompleteSuggestions(token)
     }
 
+    override fun view(tagName: String) {
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt:187]: `view()` is a bit of generic of a name, maybe `viewTag()`?

File: app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt [hunk 2]
```diff
@@ -180,6 +184,15 @@ class FollowedTagsActivity :
...
+        startActivity(StatusListActivity.newHashtagIntent(this, tagName))
+    }
+
+    override fun copyTagName(tagName: String) {
+        getSystemService(ClipboardManager::class.java).setPrimaryClip(ClipData.newPlainText(null, "#$tagName"))
+        Toast.makeText(this, getString(R.string.confirmation_hashtag_copied, tagName), Toast.LENGTH_SHORT).show()
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt:193]: modern Android anyway shows popup when something is copied, I am wondering if we need anything else

**Review by charlag — APPROVED** [review]

File: app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt (hunk 2 above)
Comment (Tak) [reply app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt:193]: Huh, it didn't for me (android 11), and we're manually showing a popup in other places, but I can remove it if newer android has us covered

Comment (cbeyls) [reply app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt:193]: From the [documentation](https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#Feedback):

Users expect visual feedback when an app copies content to the clipboard. This is done automatically for users in Android 13 and higher, but it must be manually implemented in prior versions.

Comment (Tak) [reply app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt]: I moved all the slightly different "copy text to clipboard" implementations to an `Activity` extension that puts the toast behind a version check

**Pull request merged**